### DISCUSSION TITLE
Small error in "Preventing Spam with an API" chapter

### DISCUSSION THREAD

**Discussion by MaximumRGB** [symfony/symfony#50364] [Symfony Book: The Fast Track]
https://symfony.com/doc/current/the-fast-track/en/16-spam.html
The API call response (endpoint _https://{AKISMET_KEY}.rest.akismet.com/1.1/comment-check_) is **true** if it finds a comment to be spam. However, there is no **x-akismet-pro-tip** header. So, the _getSpamScore()_ method does not return **2**, even if _comment_author='akismet-guaranteed-spam'_ or _comment_author_email='akismet-guaranteed-spam@example.com'_.

**Comment by davidoliver-1** [comment]
If you have encountered an error in a specific chapter or resource, I recommend reaching out to the provider of the material directly. They will be able to address any issues or provide clarification regarding the content you are referring to.
